Makefile

```diff
@@ -51,6 +51,10 @@ asset-page-flag: ## insert a waffle flag into local docker devstack
 i18n.docker: ## what devs should do from their host machines
 	docker exec -t dahlia.studio-frontend bash -c 'make i18n.extract && make i18n.preprocess'
 
+extract_translations: | i18n.extract ## tooling expects a target with this name
+	# every other repo is python, so we need to use this hook to npm install on jenkins
+	npm install
```

Comment:
In the future, as we have more JS-only IDAs, we might want to change the jenkins job so it knows to `npm install` for JS repos, but for now I'm alright with this.

Reply:
agreed